dumper/app/src/main/resources/oracle-stats/cdb/awr/segment-stats.sql

```diff
@@ -27,11 +27,18 @@ SELECT
   SegStat.physical_writes_delta "PhysicalWritesBlocks",
   SegStat.physical_read_requests_delta "PhysicalReadRequestsBlocks",
   SegStat.physical_write_requests_delta "PhysicalWriteRequestsBlocks",
-  SegStat.table_scans_delta "TableScans"
+  SegStat.table_scans_delta "TableScans",
+  SegStatObj.owner "ObjectOwner",
+  SegStatObj.object_name "ObjectName",
+  SegStatObj.object_type "ObjectType"
 FROM cdb_hist_seg_stat SegStat
+INNER JOIN cdb_hist_seg_stat_obj SegStatObj
+  ON SegStat.ts# = SegStatObj.ts#
+  AND SegStat.obj# = SegStatObj.obj#
+  AND SegStat.dataobj# = SegStatObj.dataobj#
 INNER JOIN cdb_hist_snapshot Snap
   ON SegStat.snap_id = Snap.snap_id
   AND SegStat.dbid = Snap.dbid
   AND SegStat.instance_number = Snap.instance_number
   -- use a query parameter to get the number of querylog days that should be loaded
-  AND Snap.end_interval_time > sysdate - ?
+  AND Snap.end_interval_time > sysdate - ?
```

Comment:
Do the newlines at the ends of this and the dba files need to be removed?

Reply:
No, returned back, as other files have new line in the end of the file

Reply:
The CI failed due to spotless errors, because there's some whitespace at the end of the last lines now - this can be removed as well